addons/l10n_it/models/account_tax.py

```diff
@@ -57,6 +57,12 @@ def _l10n_it_get_tax_kind(self):
             return 'vat'
         return None
 
+    def _l10n_it_get_legal_notes_text(self):
+        self.ensure_one()
+        if self.invoice_legal_notes:
+            return html2plaintext(self.invoice_legal_notes)
+        return ''
+
```

Comment:
```suggestion
```
Do we still need this method?

Reply:
ah yes, we don't need it now. Previously it was just because `get_text_content` raises when given `False` value, but `html2plaintext` can handle that now.